src/zoid/buttons/util.js

```diff
@@ -72,33 +72,33 @@ export function isSupportedNativeBrowser() : boolean {
     return false;
 }
 
-export function createVenmoExperiment() : Experiment | void {
-    return inlineMemoize(createVenmoExperiment, () => {
-        const enableFunding = getEnableFunding();
-        const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1;
+export function createVenmoExperiment() : ?Experiment {
+    const enableFunding = getEnableFunding();
+    const isEnableFundingVenmo = enableFunding && enableFunding.indexOf(FUNDING.VENMO) !== -1;
 
-        const fundingEligibility = getFundingEligibility();
-        const isEligibleForVenmo = fundingEligibility && fundingEligibility[FUNDING.VENMO] && fundingEligibility[FUNDING.VENMO].eligible;
+    const fundingEligibility = getFundingEligibility();
+    const isEligibleForVenmo = fundingEligibility && fundingEligibility[FUNDING.VENMO] && fundingEligibility[FUNDING.VENMO].eligible;
 
-        if (isDevice()) {
-            if (!isEligibleForVenmo || (isEnableFundingVenmo && isSupportedNativeBrowser()) || !isSupportedNativeBrowser()) {
-                return;
-            }
+    if (isDevice()) {
+        if (!isEligibleForVenmo || (isEnableFundingVenmo && isSupportedNativeBrowser()) || !isSupportedNativeBrowser()) {
```

Comment:
Also not a block but if you have some extra time and wanna refactor this into a variable, for descriptive purposes, i wouldn't complain ;)

Reply:
Great point! I pushed up an improvement.